terraform/templates/local/docker_compose_from_source.tpl

```diff
@@ -9,7 +9,7 @@ services:
 
   aws-ot-collector:
     build:
-      context: ../../../aws-otel-collector
+      context: $GOPATH/src/github.com/aws-observability/aws-otel-collector
```

Comment:
why change this? If you clone the test in the gopath way, the relative path should work.

Reply:
Why not? `$GOPATH` is more accurate than relative path.

Reply:
some people don't really have gopath there, the assumption we are making here is testing-framework code is at the same folder of collector code.  if we want to change it, please change the document also.

Reply:
Thanks for the explanation. Reverted.